src/lib/components/Notification.svelte

```diff
@@ -1,32 +1,44 @@
 <script lang="ts">
-  export let notificationTitle: string;
-  export let notificationText: string;
-  export let notificationLink: string;
+  import LiveTimeSince from "$lib/components/LiveTimeSince.svelte";
+  import type { NotificationSchema } from "$lib/zod/schemas";
+  import type { Notification } from "@prisma/client";
+  import type { SuperValidated } from "sveltekit-superforms";
+  import { superForm } from "sveltekit-superforms/client";
+  type NotificationItem = Pick<
+    Notification,
+    "link" | "title" | "message" | "createdAt" | "id"
+  >;
 
-  const handleDelete = () => {
-    console.log("Deleting notification");
-  };
+  export let notification: NotificationItem;
+  export let deleteForm: SuperValidated<NotificationSchema>;
 
-  const handleClick = () => {
-    console.log("Clicking on notification");
-  };
+  const { enhance } = superForm(deleteForm, {
+    id: notification.id.toString(),
+  });
```

Comment:
We use superForm in this project, and thus we don't use the bultin `enhance` method, feel free to check out how I solved it with this setup for future PRs